concepticondata/conceptlists/Smith-2017-680.tsv

```diff
@@ -0,0 +1,681 @@
+NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+1	above	1741	ABOVE
+2	adam's apple	802	ADAM'S APPLE
+3	aflame	1428	BURNING
+4	afraid	1419	FEAR (BE AFRAID)
+5	after	1685	AFTER
+6	age	1789	AGE
+7	agree	3643	AGREE
+8	alive	1422	BE ALIVE
+9	all	98	ALL
+10	alone	1964	ALONE
+11	already	2576	ALREADY
+12	always	1676	ALWAYS
+13	ancestors	1669	ANCESTORS
+14	and	1577	AND
+15	angry	1826	ANGRY
+16	animal	619	ANIMAL
+17	ankle	803	ANKLE
+18	annoyed	2223	BE ANGRY
+19	answer	1275	ANSWER
+20	ant	587	ANT
+21	ape, orangutan	3298	APE OR MONKEY
+22	armpit	1886	ARMPIT
+23	arrive	1387	ARRIVE
+24	ash	646	ASH
+25	ask	1309	ASK (INQUIRE)
+26	aunt	1272	AUNT
+27	baby	1289	BABY
+28	baby carrier	3926	BABY CARRIER
+29	back	1291	BACK
+30	bad	1292	BAD
+31	bait	2013	BAIT
+32	bamboo	1927	BAMBOO
+33	banana	868	BANANA
+34	banyan	346	BANYAN
+35	bark	1204	BARK
+36	barking deer	3152	MUNTJACS
+37	bathe	138	BATHE
+38	bear	1820	BEAR
+39	because	1157	BECAUSE
+40	become	1990	BECOME
+41	bee	665	BEE
+42	before	1882	BEFORE
+43	begin	3447	BEGIN OR START
+44	behind	1686	BEHIND
+45	bent	297	CROOKED
+46	betel chew	3161	CHEW BETEL
+47	betel nut	217	BETELNUT
+48	between	689	BETWEEN
+49	big	1202	BIG
+50	binturong
+51	bird	937	BIRD
+52	bite	1403	BITE
+53	bitter	887	BITTER
+54	black	163	BLACK
+55	blader	1746	BLADDER
+56	blind	1653	BLIND
+57	blink	1655	BLINK
+58	blood	946	BLOOD
+59	blow, as wind	175	BLOW (OF WIND)
+60	blow nose	2679	BLOW (WITH NOSE)
+61	blowpipe
+62	blow, with mouth	176	BLOW (WITH MOUTH)
+63	boar	1348	BOAR (MALE PIG)
+64	body	1480	BODY
+65	body hair	189	HAIR (BODY)
+66	bone	1394	BONE
+67	bored	905	BORING
+68	bowl	1530	BOWL
+69	brain	1583	BRAIN
+70	branch	1531	BRANCH
+71	brave	3	BRAVE
+72	breast	1402	BREAST
+73	breast feed	2472	SUCKLE (BREASTFEED, NURSE)
+74	breathe	1407	BREATHE
+75	bridge	1011	BRIDGE
+76	broom	245	BROOM
+77	burn	141	BURN (SOMETHING)
+78	burp	2789	BURP
+79	burry	1719	BURY
```

Comment:
Sadly, there are no specification of the gloss in the original source. I assumed it was a typo, since there are others in the source and since there is no presence of the `bury` gloss in the list or in the text of the source.

Reply:
@The-Austronesianist here's another case where we were not sure whether we are dealing with a typo.

Reply:
burry = BURY